src/components/ClientDialog.tsx

```diff
@@ -77,18 +77,7 @@ function ClientDialog(props: ClientDialogProps) {
   };
 
   const saveClientDialog = () => {
-    setBusy(true);
-    // type handle separate to support RadioButton
-    clientService.newRequest({
-      ...request,
-      // TODO not sure how we want to handle these ids, needs lookup control?
-      pet_id: null,
-      team_member_id: null,
-    }, client, animal)
-      .then((requestResponse) => props.onClose(requestResponse))
-      // TODO - handle all sorts of errors: client exists, animal exists, request exists, etc.
-      .catch((err) => props.onClose(null))
-      .finally(() => setBusy(false));
+    // TODO implement client creation
```

Comment:
how does getting rid of the client creation affect the code? Is it handled somewhere else?

Reply:
This code wasn't creating a client. It was creating a service request. It was code that was copied over from another place that has already been cleaned up. 

We  need to add the behavior for creating a client. The good thing is we now have the queries and the UI components for it (the client section), we just need to wire them up!